test/test_torch.py

```diff
@@ -1525,6 +1525,10 @@ def test_nondeterministic_alert_put(self, device):
                 lambda: op_call(a, indices, values, accumulate=False),
                 'put_')
 
+    # warn_only=False correctly raises RuntimeError: put_ does not have a deterministic implementation
+    # warn_only=True logs warning from the FallbackKernel: torch.ops.aten.put_.default, instead of as UserWarning:
+    # [W Context.cpp:%(lineno)] Warning: put_ does not have a deterministic implementation
+    @skipIfTorchInductor("warning is logged from the FallbackKernel: torch.ops.aten.put_.default when warn_only=True")
```

Comment:
Hi @jansel, skipping the following test. Warning when `warn_only=True` is correctly raised form the fallback kernel when graph break is eliminated.